flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/BoundedMultiInput.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.streaming.api.operators;
+
+import org.apache.flink.annotation.PublicEvolving;
+
+/**
+ * Interface for the multi-input operators that can process EndOfInput event.
+ */
+@PublicEvolving
+public interface BoundedMultiInput {
+
+	/**
+	 * It is notified that no more data will arrive on the input identified by the {@code inputId}.
+	 * The {@code inputId} is numbered starting from 1, and `1` indicates the first input.
```

Comment:
Should this start with `1` or `0`? I would prefer `0` unless there is some pre-existing convention in Flink.

Reply:
prefer `1`, because `TwoInputStreamOperator.processElement1` and `TwoInputStreamOperator.processElement2` not `processElement0` and `processElement1`.